src/main/java/bio/terra/folder/db/FolderDao.java

```diff
@@ -33,17 +33,8 @@ public void createFolder(
     paramMap.put("folder_name", folderName);
     paramMap.put("spend_profile_inherited", spendProfileInherited);
 
-    if (parentFolderId.isPresent() && parentFolderId.get() != null) {
-      paramMap.put("parent_folder_id", parentFolderId.get());
-    } else {
-      paramMap.put("parent_folder_id", null);
-    }
-
-    if (spendProfile.isPresent() && spendProfile.get() != null) {
-      paramMap.put("spend_profile_id", spendProfile.get());
-    } else {
-      paramMap.put("spend_profile_id", null);
-    }
+    paramMap.put("parent_folder_id", parentFolderId.isPresent() ? parentFolderId.get() : null);
+    paramMap.put("spend_profile_id", spendProfile.isPresent() ? spendProfile.get() : null);
```

Comment:
bah, I am learning about JsonNullable as I go. Amending my previous feedback, this can be even cleaner, using orElse:
```
paramMap.put("parent_folder_id", parentFolderId.orElse(null));
```

Reply:
Neat, thanks!